src/StreamingDataFrames/streamingdataframes/models/serializers/quix.py

```diff
@@ -94,20 +177,18 @@ def __call__(self, value: bytes, ctx: SerializationContext) -> Iterable[dict]:
         :return: Iterable of dicts
         """
 
-        # Validate message headers
         # Warning: headers can have multiple values for the same key, but in context
         # of this function it doesn't matter because we're looking for specific keys.
-        headers_dict = (
-            {key: value.decode() for key, value in ctx.headers}
-            if ctx.headers is not None
-            else {}
-        )
-        # Fail if the message is a part of a split
-        if Q_SPLITMESSAGEID_NAME in headers_dict:
-            raise SerializationError(
-                f'Detected "{Q_SPLITMESSAGEID_NAME}" header but message splitting '
-                f"is not supported"
-            )
+        if as_legacy := self._is_legacy_format(value):
```

Comment:
Is the `:=` operator required here? 
The `as_legacy` variable doesn't seem to be used anywhere else. 

It can be replaced with `if self._is_legacy_format(value):`

Reply:
It is used on line 225

Reply:
Oh ok, it didn't notice it so far away :)